frontend/app/electron/main/http.ts

```diff
@@ -156,3 +160,72 @@ export function stopHttp() {
   if (server?.listening)
     server.close();
 }
+
+export function startWalletConnectBridgeServer(port: number, logger: LogService) {
```

Comment:
I will need your help to check this. This is to open the bridge.
it doesn't open a single html file, but it open a route on the app itself.